source/PlayniteUI/Skins/Classic/Classic.xaml

```diff
@@ -110,12 +109,17 @@
         <Setter Property="Template">
             <Setter.Value>
                 <ControlTemplate TargetType="{x:Type Button}">
-                    <Border BorderThickness="{TemplateBinding BorderThickness}" Background="{TemplateBinding Background}"
+                    <Border x:Name="Border" BorderThickness="{TemplateBinding BorderThickness}" Background="{TemplateBinding Background}"
                             CornerRadius="{DynamicResource ControlCornerRadius}" Style="{DynamicResource HighlightBorder}">
                         <ContentPresenter HorizontalAlignment="{TemplateBinding HorizontalContentAlignment}"
                                           VerticalAlignment="{TemplateBinding VerticalContentAlignment}"
                                           Margin="{TemplateBinding Padding}" RecognizesAccessKey="True" />
                     </Border>
+                    <ControlTemplate.Triggers>
+                        <Trigger Property="IsFocused" Value="True">

```

Comment:
the HighlightBorder style has a focus trigger already but it doesn't work for the button border. <ControlTemplate TargetType="{x:Type TextBox}"> also has this same trigger on it to get the border to work (i just copy/pasted it to this style).  I'm assuming this is because the "control" has focus, not the boarder.

Reply:
I meant this for all control you are styling based on `IsFocused`. You should instead set `FocusVisualStyle` property to the style that will describe the look of control if control is focused. https://docs.microsoft.com/en-us/dotnet/api/system.windows.frameworkelement.focusvisualstyle?view=netframework-4.7.2